libs/regexp/regexp.c

```diff
@@ -311,7 +311,7 @@ static value regexp_matched_num( value o ) {
 	d = PCRE(o);
 	if( val_is_null(d->str) )
 		return alloc_int(-1);
-	return alloc_int(d->nmatches);
+	return alloc_int(d->nmatchs);
```

Comment:
I renamed this field to `n_groups` while porting because I found it confusing (and because of the typo). Changing this from `nmatchs` to `n_groups` should be the fix.